### PR TITLE
App fix music player ui

### LINKED ISSUE #628: Music Player Updates
via https://github.com/NewSpring/newspring-app/issues/186

### PR DESCRIPTION
**Pull request by samclaridge**
Many much UI improvements for the Music Player.

Fixes #628

### CODE REVIEW

File: apollos/core/blocks/sections/index.jsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default class SectionsContainer extends Component {
   componentDidMount() {
     this.props.dispatch(navActions.setLevel("TOP"))
     this.props.dispatch(modal.update({keepNav: true}))
+    this.props.dispatch(modal.update({modalBackground: "light"}))
```
Comment (johnthepink) [inline apollos/core/blocks/sections/index.jsx]: These two modal dispatches can be combined to:

```javascript
this.props.dispatch(modal.update({
  keepNav: true,
  modalBackground: "light",
});
```

File: sites/app/modules/app/client/components/players/audio/audio.Controls.jsx [hunk 2]
```diff
@@ -223,24 +263,26 @@ export default class AudioControls extends Component {
           isLight={this.props.isLight}
         />
 
-        <div className="grid one-whole flush">
+      <div className="grid one-whole flush" style={this.controlGridStyles}>
```
Comment (johnthepink) [inline sites/app/modules/app/client/components/players/audio/audio.Controls.jsx]: nit picky, but can we fix the indent here?

File: sites/app/modules/app/client/components/players/audio/audio.MiniPlayer.jsx [hunk 3]
```diff
@@ -218,7 +218,7 @@ export default class MiniPlayer extends Component {
             className={this.albumClasses()}
             style={bgImageStyle}
           ></div>
-          <div className="plain floating__item">
+        <div className="plain floating__item six-eighths truncate">
             <h6 className="display-block text-dark-secondary flush">
```
Comment (johnthepink) [inline sites/app/modules/app/client/components/players/audio/audio.MiniPlayer.jsx]: indent :trollface:

File: sites/app/modules/app/client/mixins/mixins.Likeable.jsx [hunk 4]
```diff
@@ -15,6 +15,7 @@ const Likeable = {
 
     if (!Meteor.userId()) {
       this.props.dispatch(modal.render(OnBoard, { coverHeader: true }))
+      this.props.dispatch(modal.update({modalBackground: "light"}))
```
Comment (johnthepink) [inline sites/app/modules/app/client/mixins/mixins.Likeable.jsx]: This is what was causing the header to still be displayed when hitting "like" and triggering the sign up form. We need to combined these dispatches to:

```javascript
this.props.dispatch(modal.render(Onboard, {
  coverHeader: true,
  modalBackground: "light",
});
```

**Comment by johnthepink** [comment]
@samclaridge nice work! 😸 Just a couple things to switch out in the inline comments.

**Comment by samclaridge** [comment]
Update for you @johnthepink

File: sites/app/modules/app/client/components/players/audio/audio.Controls.jsx (hunk 2 above)
Comment (johnthepink) [inline sites/app/modules/app/client/components/players/audio/audio.Controls.jsx]: :trollface:

**Comment by johnthepink** [comment]
🎉

**Pull request merged**